### PR TITLE
Add @ferhatelmas as a maintainer

### PR DESCRIPTION
**Pull request by ferhatelmas**

### CODE REVIEW

File: config/maintainers.json [hunk 1]
```diff
@@ -20,6 +20,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": "Front-end web developer with years of experience using JavaScript, but learning a functional approach to web development using PureScript and Haskell"
+    },
+    {
+      "github_username": "ferhatelmas",
+      "alumnus": false,
+      "show_on_website": true,
```
Comment (SleeplessByte) [inline config/maintainers.json]:
```suggestion
      "show_on_website": true,
```

cc @ErikSchierboom see if ya want @ferhatelmas as contributing or senior

Comment (ErikSchierboom) [reply config/maintainers.json]: I'll defer to @iHiD :D

**Comment by iHiD** [comment]
All maintainers starting on tracks should start as "Contributing Maintainers" for now.

I'm going to actually encode this logic into the website for v3, so the flag will be a preference, and the actual resolution of it is based on activity, but for now, I think this should be `false`. Please :)

File: config/maintainers.json (hunk 1 above)
Comment (iHiD) [inline config/maintainers.json]:
```suggestion
      "show_on_website": false,
```

Comment (ferhatelmas) [reply config/maintainers.json]: I copied my config on go, should I update there too?

Comment (iHiD) [reply config/maintainers.json]: No, you've been around Go long enough to be considered a "Senior Maintainer" there :)

The aim with this is that people contribute to a track a bit **before** they get the same credit on the website that other maintainers on that track get.

Comment (ferhatelmas) [reply config/maintainers.json]: Sounds good, thanks!

**Pull request merged**